### ISSUE TITLE
How to set parameters when I use my own data ?

### ISSUE THREAD

**Issue by Hira8023** [OSU-BMBL/scDEAL#8]
Hi,
When I use my own single cell data, is there any basis for parameter settings?
Including: bottleneck, encoder dimensions, predictor dimensions, encoder model, dropout, learning rate, and sampling.

**Comment by juychen** [comment]
Hi, we have only tested the pre-built data at the moment.  For your own data you may refer to the list of params on the github

**Comment by libmelo** [comment]
> Hi, we have only tested the pre-built data at the moment. For your own data you may refer to the list of params on the github

Hi, I have some suggestions. The scripts may need to be explained more clearly. For example, the scripts should clarify their roles, necessary inputs and outputs (specify which params for inputs, outputs and different modes). Besides, the formats of inputs and outputs may also need to be explained in details (gene symbl or id, normalization etc.).

**Comment by SiyuanHuang1** [comment]
I completely agree with the above point of view. I spent half a day configuring the environment and reading the documentation, but I still don’t understand how to use it. In addition, after reading other issues, it seems that no one has been able to successfully apply this software to their own data. I hope the author can improve it well. Thank you in advance.
